Makefile

```diff
@@ -70,22 +70,6 @@ conf/ssl/server.crt:
 bin/pfcmd: src/pfcmd
 	cp src/pfcmd bin/pfcmd
 
-.PHONY:sudo
-
-sudo:
```

Comment:
I'd leave this so that it copies the file to the proper place.

This way features requiring different sudo rights will be ok when making make devel